### PR TITLE
Added functionality to scaffold MVC View Components

### PR DESCRIPTION
**Pull request by larsw**
What the title says.

Partially solves https://github.com/OmniSharp/generator-aspnet/issues/50

### CODE REVIEW

File: templates/mvcviewcomponent.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNet.Mvc;
+
+namespace <%= namespace %>
+{
+    public class <%= classname %> : ViewComponent
+    {
+		public IViewComponentResult Invoke()
```
Comment (nosami) [inline templates/mvcviewcomponent.cs]: the indent hurts my eyes!

Comment (larsw) [inline templates/mvcviewcomponent.cs]: Bah, sorry - I actually hacked it together with Notepad2 - thought I had fixed the indentation :-/

**Comment by sayedihashimi** [comment]
I need to have some from the aspnet team to review this. @DamianEdwards can you, or someone on the team review?

**Comment by spboyer** [comment]
PR is stalled, closing as there has been no comments or further discussion on need.  Will revisit if more discussion on #50

**Pull request closed without merging**